invoker/core/src/test/java/com/google/cloud/functions/invoker/BackgroundFunctionExecutorTest.java

```diff
@@ -62,4 +70,38 @@ public void backgroundFunctionTypeArgument_raw() {
         (Class<? extends BackgroundFunction<?>>) (Class<?>) ForgotTypeParameter.class;
     assertThat(backgroundFunctionTypeArgument(c)).isEmpty();
   }
+
+  @Test
+  public void parseLegacyEventPubSub() throws IOException {
+    try (Reader reader = new InputStreamReader(getClass().getResourceAsStream("/pubsub_background.json"))) {
+      Event event = BackgroundFunctionExecutor.parseLegacyEvent(reader);
+
+      Context context = event.getContext();
+      assertEquals("google.pubsub.topic.publish", context.eventType());
```

Comment:
I think you should
```
import static com.google.common.truth.Truth.assertThat;
```
in addition to the existing `Truth8.assertThat` import, and then write
```
assertThat(context.eventType()).isEqualTo("google.pubsub.topic.publish");
```
That would be consistent with existing tests. Obviously the same applies to the other `assertEquals` calls.